src/player_hardcoded_effects.cpp

```diff
@@ -370,6 +370,9 @@ static void eff_fun_hot( player &u, effect &it )
             { { bp_foot_r, 2 }, { 0, 0, 0, 0, "", 0, "" } },
         }
     };
+    // Maximum chance of 1 in MIN_EFFECTIVE_TEMPERATURE to vomit
+    const int MIN_EFFECTIVE_TEMPERATURE = 25;
```

Comment:
Please just use the number directly. A name for it is not needed, it's used exactly once in a very specific context (which uses other magic numbers anyway). And it makes the expression much longer and more difficult to understand.

Reply:
Understandable. Figured it being a named constant would be easier to handle if the value needed to be tuned up or down. Modification applied.